v1.1/deploy-cockroachdb-on-aws-insecure.md

```diff
@@ -252,9 +252,11 @@ To test this, install CockroachDB locally and use the [built-in SQL client](use-
 
 4. Check which node you were redirected to:
```

Comment:
I think you can write this entire section just once in e.g. `somesteps.md` then reuse it in each of the DC tutorials via `{% include %}`. If that is not possible, let's investigate together how to make it possible. This will save editing work in the future!